semanticdb/metacp/src/main/scala/scala/meta/metacp/Settings.scala

```diff
@@ -69,8 +73,8 @@ object Settings {
       args match {
         case "--" +: rest =>
           loop(settings, false, rest)
-        case "--cache-dir" +: cacheDir +: rest if allowOptions =>
-          loop(settings.copy(cacheDir = AbsolutePath(cacheDir)), true, rest)
```

Comment:
Shall we maybe emit a more informative error here?

Reply:
Good idea, done. Metacp now reports "--cache-dir is deprecated, use --out instead"